src/NUnitTestAdapter/NUnit3TestDiscoverer.cs

```diff
@@ -1,6 +1,25 @@
-﻿// ****************************************************************
-// Copyright (c) 2011-2015 NUnit Software. All rights reserved.
-// ****************************************************************
+﻿// ***********************************************************************
+// Copyright (c) 2011-2015 Charlie Poole
```

Comment:
Outside the scope of this pull request, but I was reading Clean Code and the recommendation is to not put these copyright headers in every file since they are just noise that forces you to scroll and they are redundant with the LICENSE file in the root. Makes sense to me. Maybe something for future discussion?